.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,30 @@
-> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.
 
-### What does this PR do?
+<!-- 
+Please use one of the following prefixes for the title:
+docs: Documentation not including procedures. Engineering review is mandatory.
+procedures: Documentation including procedures. Testing procedures is mandatory. Engineering and QE review is mandatory (Engineering can review on behalf of QE). 
+chore: Choreography, release, tooling, version upgrades.
```

Comment:
What does Choreography mean here?

Reply:
```suggestion
chore: Routine, release, tooling, version upgrades.
```

You are right, chore is not the short form of choreography here. The conventionalcommit word for routines.